tests/integration/selinux.bats

```diff
@@ -46,8 +46,8 @@ function teardown() {
 	[ "$status" -eq 0 ]
 }
 
-@test "runc run (custom selinux label, RUNC_DMZ=legacy)" {
-	export RUNC_DMZ=legacy
+@test "runc run (custom selinux label, RUNC_DMZ=false)" {
```

Comment:
ditto

Reply:
idem. And next time don'y worry, I can `git grep` :)